src/helics/core/zmq/ZmqCommsCommon.cpp

```diff
@@ -46,5 +46,12 @@ bool bindzmqSocket (zmq::socket_t &socket,
     return bindsuccess;
 }
 
+std::string getZMQVersion ()
+{
+    auto vers = zmq::version ();
+    return std::string ("ZMQ v") + std::to_string (std::get<0> (vers)) + '.' +
+           std::to_string (std::get<2> (vers)) + '.' + std::to_string (std::get<0> (vers));
```

Comment:
Should the element from the version accessed be 1 and 2 for the minor and patch numbers?
```suggestion
           std::to_string (std::get<1> (vers)) + '.' + std::to_string (std::get<2> (vers));
```